samples/tab-deeplink/csharp/README.md

```diff
@@ -69,6 +69,9 @@ urlFragment: officedev-microsoft-teams-samples-tab-deeplink-csharp
     git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
```

Comment:
Do we need to update all the screenshots as well?

Reply:
Updated images and gif's